PWG/DevNanoAOD/AliAnalysisNanoAODCuts.cxx

```diff
@@ -120,13 +120,17 @@ void AliNanoAODSimpleSetter::SetNanoAODHeader(const AliAODEvent * event   , AliN
 
   TString firedTriggerClasses = header->GetFiredTriggerClasses();
 
-  static const char * validatorString[] = {"Centr","CentrTRK","CentrCL0","CentrCL1", "MagField", "OfflineTrigger", "RunNumber", "FiredTriggerClasses", 0};
+  UShort_t bunchCrossNumber = header->GetBunchCrossNumber();
+  UInt_t orbitNumber = header->GetOrbitNumber();
+  UInt_t periodNumber = header->GetPeriodNumber();
+
+  static const char * validatorString[] = {"Centr","CentrTRK","CentrCL0","CentrCL1", "MagField", "OfflineTrigger", "RunNumber", "FiredTriggerClasses", "BunchCrossNumber", "OrbitNumber", "PeriodNumber", 0};
```

Comment:
Why static, and why const char *? Better use array of const TStrings (in particular const std::array< const TString, ...>), non-static.